packages/babel-plugin-material-ui/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@material-ui/babel-plugin-material-ui",
```

Comment:
```suggestion
  "name": "@material-ui/babel-plugin-optimize-material-ui",
```

The current name suggests the plugin is required.

Reply:
In that case, should we still keep the scope? (`@material-ui/`)

Reply:
Oh didn't even spot that. Yeah we don't need this in any case.